src/flb_engine.c

```diff
@@ -694,6 +695,10 @@ int flb_engine_start(struct flb_config *config)
             flb_sched_timer_cleanup(config->sched);
             flb_upstream_conn_pending_destroy_list(&config->upstreams);
         }
+
+        if (is_error_reporting_enabled()) {
+            flb_info("[engine] error reporting ttl fresh");
+        }
```

Comment:
One thing I am thinking. This feature is not something most folks will want. I think we should create cmake build flag for it and disable it by default. Then in our build of fluent bit we can enable it.

Reply:
added the flag: FLB_AWS_ERROR_REPORTER and default not have this to build.